packages/shared/src/components/cards/PostCard.tsx

```diff
@@ -69,12 +73,17 @@ export const PostCard = forwardRef(function PostCard(
     children,
     showImage = true,
     style,
+    flags,
     ...props
   }: PostCardProps,
   ref: Ref<HTMLElement>,
 ): ReactElement {
   const [selectedComment, setSelectedComment] = useState<Comment>();
   const { trending } = post;
+  const postHeadingFont = getThemeFont(
```

Comment:
I think it might be better to load it in the Feed.tsx file and then pass down to FeedItemComponent as a option?
That way we only have to load it once.